### PR TITLE
[README.md]Fix typos in mmaction dev-1.x

### PR DESCRIPTION
**Pull request by Zheng-LinXiao**
## Motivation

Improve the readme.md of mmaction2 dev-1.x

## Modification
The Link in README.md is wrong.
Fix the link of [📘Documentation](https://mmaction2.readthedocs.io/en/1.x/)  to [📘Documentation](https://mmaction2.readthedocs.io/en/dev-1.x/index.html)
Fix the link of [🛠️Installation](https://mmaction2.readthedocs.io/en/1.x/get_started.html) to [🛠️Installation](https://mmaction2.readthedocs.io/en/dev-1.x/get_started.html#installation)
Fix the link of [🆕Update News](https://mmaction2.readthedocs.io/en/1.x/notes/changelog.html) to [🆕Update News](https://mmaction2.readthedocs.io/en/dev-1.x/notes/changelog.html)
Fix the install.md link in the Installation section of README.md
Fix the link of https://mmaction2.readthedocs.io/en/1.x/get_started.html to https://github.com/open-mmlab/mmaction2/blob/dev-1.x/docs/en/get_started.md

## Checklist

1. Pre-commit or other linting tools should be used to fix the potential lint issues.
2. The modification should be covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation should be modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by ly015** [comment]
Thank you very much for your contribution. It's inspiring to receive feedback on the newly released 1.0 version. We have enabled the readthedocs pages for the 1.x branch so '1.x' in the URLs should be okay now. Other fixes are still necessary and we will help check the links.

File: README.md [hunk 1]
```diff
@@ -17,10 +17,10 @@
     </sup>
   </div>
 
-[📘Documentation](https://mmaction2.readthedocs.io/en/1.x/) |
-[🛠️Installation](https://mmaction2.readthedocs.io/en/1.x/get_started.html) |
+[📘Documentation](https://mmaction2.readthedocs.io/en/dev-1.x/index.html) |
```
Comment (hukkai) [inline README.md:20]: The 1.x branch is a released branch while dev-1.x branch is a developing branch that may change every day. So I think it should be https://mmaction2.readthedocs.io/en/1.x/

File: README.md [hunk 2]
```diff
@@ -17,10 +17,10 @@
...
+[🛠️Installation](https://mmaction2.readthedocs.io/en/dev-1.x/get_started.html#installation) |
```
Comment (hukkai) [inline README.md:21]: The installation also requires the #PREREQUISITES part

**Review by hukkai — COMMENTED** [review]
These typos seems not typos to me, can you confirm @ly015 ?

File: README.md (hunk 1 above)
Comment (Zheng-LinXiao) [reply README.md:20]: Thank you very much for your reply. Now the readthedocs page has been opened. The problems I raised have been solved

**Comment by Zheng-LinXiao** [comment]
Thank you very much for your reply. Now the readthedocs page has been opened. The problems I raised have been solved. I also want to ask, is it because the ckpt has not been updated yet? It seems that 1. x or dev-1. x does not correspond to each other (specifically, I am at: https://github.com/open-mmlab/mmaction2/discussions/1929 There are questions), and then the ckpt in some projects is empty. Because I don't understand the relationship between them, I will report an error when verifying, so I didn't mention the relevant pr.

**Comment by ly015** [comment]
Hi, thanks for this PR. We will close it for now and feel free to contact us if there are any other questions.

**Pull request closed without merging**